typescript/lib/bulk_command_execution/v0_bulk_command_api/v0_command_types.ts

```diff
@@ -0,0 +1,72 @@
+import { ok, err, Result } from "neverthrow";
+
+// We provide a visitor interface here so that:
+//  1) all enum cases can be exhaustively handled
+//  2) any changes in the enum will result in a compile break
+export interface V0CommandTypeVisitor {
+	visitLoadLambda: () => Result<null, Error>;
+	visitExecuteLambda: () => Result<null, Error>;
+	visitRegisterService: () => Result<null, Error>;
+	visitGenerateFiles: () => Result<null, Error>;
+	visitLoadStaticFiles: () => Result<null, Error>;
+	visitStartService: () => Result<null, Error>;
+	visitRemoveService: () => Result<null, Error>;
+	visitRepartition: () => Result<null, Error>;
+	visitExecCommand: () => Result<null, Error>;
+	visitWaitForEndpointAvailability: () => Result<null, Error>;
+	visitExecuteBulkCommands: () => Result<null, Error>;
+}
+
+export type V0CommandType = string;
+
+// vvvvvvvvvvvvvvvvvvvv Update the visitor whenever you add an enum value!!! vvvvvvvvvvvvvvvvvvvvvvvvvvv
+const loadLambdaCommandType: V0CommandType = "LOAD_LAMBDA";
```

Comment:
Golang doesn't have enums so we had to do it this way, but Typescript does have enums so you should use an enum here